INSTALL.md

```diff
@@ -8,7 +8,7 @@ How to install application using helm:  <br />
     helm install ReleaseName ChartName
     
     a.) Add helm repository in tractusx:
-           helm repo add autosetup https://eclipse-tractusx.github.io/charts/dev
+           helm repo add autosetup https://eclipse-tractusx.github.io/charts/tractusx-dev
```

Comment:
```suggestion
           helm repo add tractusx-dev https://eclipse-tractusx.github.io/charts/tractusx-dev
```

Reply:
Earlier we used few versions in middle for testing purpose so the plan was for 1.2.8, but i discussed with Sigi , he said we have to bump a version and should be in in sync with tractus-x repo versions. We should not put gap in between the versions. 
Last version became 1.2.5, so we modified this as 1.2.6